spectests/ssz_spec_test.go

```diff
@@ -0,0 +1,811 @@
+package autogenerated
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/bazelbuild/rules_go/go/tools/bazel"
+	"github.com/ghodss/yaml"
+	"github.com/prysmaticlabs/go-ssz"
+)
+
+func TestYamlSpecTests(t *testing.T) {
+	topPath := "/eth2_spec_tests/tests/ssz_static/core/"
```

Comment:
This is only the core files? What about ssz_generic/uint? Maybe for another PR, but we are missing some spec tests.